models/service.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import "math/rand"
+
+type Service struct {
```

Comment:
Ok, I see this, so perhaps we have a mix here as I guess some of these models will be provided by the API itself.
I have an open question, perhaps it is not necessary to respond now and here, but I guess it's related.

The model we use has a lot of data that probably we won't needed, so a first possibility is to define here the "REST" model where we map a subset of the all model used querying the API, or a second possibility, we can just forward the model used by the backend clients, which is mostly prepared with the json annotations.

So, first option has "pros" that we map only what we need with the "cons" of extra cost mappings, and second option we just forward models we fetch but probably we won't use all info, so response will be more heavy, but on the contrary, if the UI needs additional data it won't require any modification on this layer.

Reply:
+1 for first option (mapping model). Provide to the UI just what it needs. Another advantage is that any breaking change in the backends would not impact the UI, just the intermediate server-side service.

Reply:
+1 on have our model representation, as well. This will gave us more power on add behavior into it.
@jotak +1